### ISSUE TITLE
1.8.1 writtes message to error stream on installation

### ISSUE THREAD

**Issue by pascalberger** [yeoman/yo#432]
Following up to #430. Current version still seems to write messages to the error stream, which might cause trouble if run in a scripted environment, which watches the error stream and considers everything written to it as an error. 

```
> yo@1.8.1 tabtab C:\Users\pascal.berger\AppData\Roaming\npm\node_modules\yo
> [ $CI ] || tabtab install --name=yo --completer=yo-complete

'[' is not recognized as an internal or external command,
operable program or batch file.
  tabtab:commands Windows shell completion not yet supported, skipping completion install +0ms
```

**Comment by sindresorhus** [comment]
> which watches the error stream and considers everything written to it as an error.

Then that environment is faulty. Exit codes indicate failure. The error stream is just an alternative stream.

**Comment by SBoudrias** [comment]
In that case though, it's just the user mis-describing his issue. It doesn't have anything to do with the error stream.

The process is down straight failing because it tries to run bash code on windows. - Time to update your windows!

To be fair, we should remove the bash part of it as it'll fail on every old windows and tabtab should check for the CI variable internally.

cc @mklabs

**Comment by pascalberger** [comment]
Concrete problem is when using in a [Chocolatey](https://chocolatey.org) package, where we just call `npm install -g yo@1.8.1` in the `chocolateyInstall.ps1` PowerShell script during the installation. Environment is Windows 10.

**Comment by mklabs** [comment]
I agree with Sindre on this one, stderr is often used as an alternative stream (debug, npmlog, etc.)

@SBoudrias Ok on checking CI variable within tabtab. Might as well add a few more env var hook to skip tabtab install or specify a meaningfull default (like `TABTAB_DEFAULT=2` or similar to skip prompts).

**Comment by SBoudrias** [comment]
@mklabs I think we should skip prompt by default.

Then maybe if a user run `yo completion` manually, we prompt them if tabtab is not installed?

**Comment by mklabs** [comment]
@SBoudrias yeah ok, was a concern when I initially did the PR. Will prepare a PR to move the `tabtab install` part behind `yo completion`.  I didn't cause it meant altering the generator loading for one keyword "completion".

**Comment by SBoudrias** [comment]
Should now be fixed in 1.8.2
